app/src/main/java/com/hammer/talkbbokki/data/repository/BookmarkRepositoryImpl.kt

```diff
@@ -110,9 +111,23 @@ internal class BookmarkRepositoryImpl @Inject constructor(
                 )
             )
         )
-    }
+    }*/
+
+    override fun removeBookmark(id: Int) = flow { emit(dao.removeBookmark(id)) }
 
-    override fun removeBookmark(id: Int) {
-        dao.removeBookmark(id)
+    override fun addBookmark(item: TopicItem) = flow {
+        emit(
+            dao.addBookmark(
+                TopicItem(
+                    id = 8,
+                    name = "너무 애틋했던 나의 옛 사랑에게서 갑자기 카톡이 왔다. 설레는 마음에 본 카톡, 환승연애 출연하자고 한다. 나간다 vs 쌍욕한다.",
```

Comment:
🚫 **[ktlint]** <max-line-length> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Exceeded max line length (100) (cannot be auto-corrected)